src/index.js

```diff
@@ -29,3 +29,4 @@ export { default as Collapse } from './components/Collapse';
 export { default as ExpansionPanel } from './components/ExpansionPanel';
 export { default as Accordion } from './components/Accordion';
 export { default as Avatar } from './components/Avatar';
+export { default as Tooltip, UncontrolledTooltip } from './components/Tooltip';
```

Comment:
Wrong component name here, I think I prefer `UncontrolledTooltip` over `EasyRichTooltip`, but I'm ok with either 🚀

Reply:
We have the `Easy*` prefixed used in other components. It's simply a pattern we've adopted for Rover. Not attached to it, but worth considering

Reply:
And yes, I'll fix the export name :)